cmd/crank/main.go

```diff
@@ -60,22 +59,9 @@ var cli struct {
 	Version versionFlag `short:"v" name:"version" help:"Print version and quit."`
 	Verbose verboseFlag `name:"verbose" help:"Print verbose logging statements."`
 
-	Install installCmd `cmd:"" help:"Install Crossplane packages."`
-	Update  updateCmd  `cmd:"" help:"Update Crossplane packages."`
-
 	XPKG xpkg.Cmd `cmd:"" help:"Crossplane package management."`
 
-	Marketplace marketplace.Cmd `cmd:"" help:"Interact with the Upbound Marketplace."`
-
-	Alpha struct {
```

Comment:
did you want to leave this alpha struct in for placeholder for the upcoming `describe`/`diagnose` commands?

Reply:
Not really. 😄 I think the beta pattern (where the command lives in a subdir) is preferable, and should be easy to follow when we add an alpha command.